nike_bot.py

```diff
@@ -1,63 +1,63 @@
-if __name__ == "__main__":
-	import requests
-	import time
-	import twitter
+#if __name__ == "__main__":
+#	import requests
+#	import time
+#	import twitter

```

Comment:
all the necessary dependencies, to get them, you have to run `pip install requests time twitter`